src/wallet/test/wallet_test_fixture.h

```diff
@@ -21,7 +21,7 @@ struct WalletTestingSetup : public TestingSetup {
     explicit WalletTestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
 
     std::unique_ptr<interfaces::Chain> m_chain = interfaces::MakeChain(m_node);
-    std::unique_ptr<interfaces::WalletClient> m_wallet_client = interfaces::MakeWalletClient(*m_chain, *Assert(m_node.args));
+    std::unique_ptr<interfaces::WalletClient> m_wallet_client = interfaces::MakeWalletClient(m_node, *m_chain, *Assert(m_node.args));
```

Comment:
Same as the comment given in src/wallet/test/init_test_fixture.cpp -> shouldn't it be an address to m_node that needs to be passed to MakeWalletClient(NodeContext, Chain, ArgsManager) method?